python/paddle/v2/fluid/layers/control_flow.py

```diff
@@ -440,9 +440,23 @@ def topk(input, k):
 
 
 def lod_tensor_to_array(x, table):
-    """
-    This function creates an operator to convert an LOD_Tensor to
-    an array.
+    """This function performs the operation that converts an LOD_Tensor to
+       an array.
+
+    Args:
+        x (LoDTensor): The tensor that needs to be converted to an array.
+        table (LoDRankTable): The RankTable that provides the coarse lod
+                              infomation to build the output LoDTensorArray.
+
+    Returns:
+        Variable: The tensor variable of type LOD_TENSOR.
+
+    Examples:
+        .. code-block:: python
+
+        x = fluid.layers.data(name='x', shape=[10])
```

Comment:
I think for statements in Line457-459, there should be another level of indentation (1 level deeper than the ".. code-block:: python) for proper rendering. Can you check?

Reply:
Yeah it was a formatting issue. Thanks for noticing this!